src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/ManagedDebuggerImageTypeService.cs

```diff
@@ -35,10 +36,11 @@ public ImageClrType TargetImageClrType
             var configuration = await _properties.GetConfigurationGeneralPropertiesAsync()
                                                  .ConfigureAwait(false);
 
-            string outputType = (string)await configuration.OutputType.GetValueAsync()
+
+            PageEnumValue outputType = (PageEnumValue)await configuration.OutputType.GetValueAsync()
```

Comment:
You should probably cast this to just `IEnumValue` instead of the concrete type itself in case it changes in the future.